src/main/java/io/cdap/plugin/google/common/GoogleAuthBaseConfig.java

```diff
@@ -47,6 +51,11 @@
   public static final String REFRESH_TOKEN_LABEL = "Refresh token";
   public static final String ACCOUNT_FILE_PATH = "accountFilePath";
   public static final String DIRECTORY_IDENTIFIER = "directoryIdentifier";
+  public static final String NAME_SERVICE_ACCOUNT_TYPE = "serviceAccountType";
+  public static final String NAME_SERVICE_ACCOUNT_JSON = "serviceAccountJSON";
+  public static final String SERVICE_ACCOUNT_FILE_PATH = "filePath";
+  public static final String SERVICE_ACCOUNT_JSON = "JSON";
+  public static final String SCHEMA = "schema";
```

Comment:
where is this used?

Reply:
`SCHEMA` is used in deserialization of config: 

https://github.com/data-integrations/google-drive/blob/3074341092eecd2522ea5ce9ac1c40877e0fc03f/src/main/java/io/cdap/plugin/google/drive/source/GoogleDriveSourceConfig.java#L358-L360